pkgs/top-level/all-packages.nix

```diff
@@ -26848,7 +26848,14 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) AppKit;
   };
 
-  btop = darwin.apple_sdk_11_0.callPackage ../tools/system/btop { };
+  inherit ({
+    btop = darwin.apple_sdk_11_0.callPackage ../tools/system/btop { };
+    btop-rocm = callPackage ../tools/system/btop {
+      rocmSupport = true;
+    };
+  })
+  btop
+  btop-rocm;
```

Comment:
```suggestion
  btop = darwin.apple_sdk_11_0.callPackage ../tools/system/btop { };

  btop-rocm = btop.override { rocmSupport = true; };
```

Reply:
The inherit is required by the CI:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#recommendation-for-new-packages-with-multiple-versions